### PR TITLE
ASC-634-Primary contact changes for organization

### PR DESCRIPTION
**Pull request by AkashExcelra**
- Updated primary contact changes feature for admin and org_admin
- Remove user-role create function from route.tsx in organization
- Fix primary contact in Landing page for edit button
- Enable edit button for admin and org_admin in Heading.tsx

### CODE REVIEW

File: apps/web/components/Organization/ListOrganization.tsx [hunk 1]
```diff
@@ -44,12 +44,13 @@ export default function ListOrganization({ userData, actionsEnabled }: ListOrgan
...
   const fetchOrganizations = async () => {
-    const organization = await getOrganization(
+    let organization = await getOrganization(
       {
         withRelation: ['orgUser', 'user_role'],
         withCount: ['projects']
       });
-
+    organization = organization.filter
+      ((organization: OrganizationDataFields) => organization.type !== 'Internal');
```
Comment (himanshubexcelra) [inline apps/web/components/Organization/ListOrganization.tsx:53]: For 'Internal' and 'external' please set enum type

File: apps/web/app/dashboard/page.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default async function Dashboard() {
       svgWidth: 28,
       svgHeight: 28,
       href: "",
-      type: myRoles.includes('admin') ? "Admin:" : "Customer Organization:"
+      type: "Admin:"
```
Comment (himanshubexcelra) [inline apps/web/app/dashboard/page.tsx:32]: Why this is changed?

File: apps/api/app/v1/organization/route.tsx [hunk 3]
```diff
@@ -297,20 +297,6 @@ export async function PUT(request: Request) {
       },
     });
 
-    if (orgAdminId !== oldAdmin) {
```
Comment (himanshubexcelra) [inline apps/api/app/v1/organization/route.tsx:300]: Why this is removed?

File: apps/web/components/Organization/editOrganization.tsx [hunk 4]
```diff
@@ -18,6 +18,7 @@ import { delay } from "@/utils/helpers";
 import { DELAY, status } from "@/utils/constants";
```
Comment (himanshubexcelra) [inline apps/web/components/Organization/editOrganization.tsx:18]: eslint max rule is not added in this file.

File: apps/web/components/Organization/ListOrganization.tsx (hunk 1 above)
Comment (AkashExcelra) [reply apps/web/components/Organization/ListOrganization.tsx]: Issue is resolved

File: apps/web/components/Organization/editOrganization.tsx (hunk 4 above)
Comment (AkashExcelra) [reply apps/web/components/Organization/editOrganization.tsx:17]: eslint max rule is added in this file.

File: apps/api/app/v1/organization/route.tsx (hunk 3 above)
Comment (AkashExcelra) [reply apps/api/app/v1/organization/route.tsx:311]: Discussed with Forum, No more need to remove roles of existing user

File: apps/web/app/dashboard/page.tsx (hunk 2 above)
Comment (AkashExcelra) [reply apps/web/app/dashboard/page.tsx:31]: Discussed with Forum, Breadcrumbs was wrong

File: apps/web/components/Dashboard/LandingPage.tsx [hunk 5]
```diff
@@ -36,7 +36,7 @@ export default function LandingPage({ userData,
     };
 
     useEffect(() => {
-        if (myRoles.includes('org_admin')) fetchOrganizationData();
+        if (myRoles.includes('admin') || myRoles.includes('org_admin')) fetchOrganizationData();
```
Comment (himanshubexcelra) [inline apps/web/components/Dashboard/LandingPage.tsx:39]: use this line here ['admin', 'org_admin'].some((role) => myRoles?.includes(role))

File: apps/web/components/Organization/ListOrganization.tsx [hunk 6]
```diff
@@ -1,4 +1,3 @@
-/*eslint max-len: ["error", { "code": 100 }]*/
```
Comment (himanshubexcelra) [inline apps/web/components/Organization/ListOrganization.tsx:1]: Why this line you are removing?

File: apps/web/components/Organization/editOrganization.tsx (hunk 4 above)
Comment (himanshubexcelra) [reply apps/web/components/Organization/editOrganization.tsx:17]: on top it's showing in red. Means you are removing

File: apps/web/components/Dashboard/LandingPage.tsx (hunk 5 above)
Comment (AkashExcelra) [reply apps/web/components/Dashboard/LandingPage.tsx]: Code is fixed, issue resolved

File: apps/web/components/Organization/ListOrganization.tsx (hunk 6 above)
Comment (AkashExcelra) [reply apps/web/components/Organization/ListOrganization.tsx]: eslint max rule is added in this file.

**Pull request merged**